apps/civil/civil-orchestrator-service/perftest.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: civil-orchestrator-service
+  namespace: civil
+spec:
+  values:
+    java:
+      keyVaults:
+        civil:
+          resourceGroup: civil
+          secrets:
+            - civil-idam-client-secret
```

Comment:
Also, any reason why you are overriding secrets in every environment ?

Reply:
it was added to get access to civil vault but i have removed it as initially we may not need it. .have pushed it